swap/swap.go

```diff
@@ -710,7 +711,7 @@ func (s *Swap) bindToContractAt(address common.Address) (contract.Contract, erro
 	if err := s.chequebookFactory.VerifyContract(address); err != nil {
 		return nil, fmt.Errorf("contract validation for %v failed: %v", address.Hex(), err)
 	}
-	swapLog.Info("bound to chequebook", "chequebookAddr", address)
+	swapLog.Info("bound to chequebook", "chequebookAddr", address, "swap_action", "bind_contract")
```

Comment:
If we would set `swap_action` with the `logger.SetLogAction`, `swap_action` would appear before `chequebookAddr` and not after, as is now the case. I would like to see it at the same place if we would set it via `logger.SetLogAction`